uiowa/uiowa
# Redirects don't work with trailing slash in source path

Closes #3979

Before running composer install, if you go to the redirect configuration (exhttps://africanamericanstudies.local.drupal.uiowa.edu/admin/config/search/redirect/add), and add a link with a trailing slash (/asdf/) to the path* field, and then add where you'd like the redirect to point to (/about), if you go to /asdf/ or /asdf, it should take you to the '/about' URL


After running composer install, if you add a link with a trailing slash (/asdf/) to the path, and then add where you'd like the redirect to point to, if you go to /asdf/ it will redirect you to what you configured - ie the correct way


# How to test


1. checkout branch
2. composer install
3. sync a site (for instance, default)
4. Go to redirect config (https://default.local.drupal.uiowa.edu/admin/config/search/redirect)
5. Add a redirect path with a trailing "/" (ie add "asdf/") and redirect to the about page
6. When you save, make sure trailing slash is removed from redirect. 
7. Go to https://default.local.drupal.uiowa.edu/asdf and https://default.local.drupal.uiowa.edu/asdf/ and make sure b oth redirect to about node
